util/src/main/java/org/kaazing/gateway/util/InternalSystemProperty.java

```diff
@@ -129,6 +128,11 @@
     GATEWAY_IDENTIFIER
             ("org.kaazing.gateway.server.GATEWAY_IDENTIFIER", ""),
 
+    // Internal system property checking if a newer version of the Gateway is available
+    UPDATE_CHECK("org.kaazing.gateway.server.UPDATE_CHECK", "true"),
+
+    SERVICE_URL("org.kaazing.gateway.server.SERVICE_URL"),
```

Comment:
I used service url as a property because it is difficult to test the gateway behaviour. The value is hard-coded in UpdateCheckGatewayObserver class. So in the integration test for the update check property, when I start the gateway I can't set the url and the real one will be access. This is the reason I used a new property. 

Also I think it's useful to have the property, if for any reasons the url will be modify the client can set the value in configuration and don't have to wait for a new release.